base/tps/src/org/dogtagpki/server/tps/processor/TPSPinResetProcessor.java

```diff
@@ -49,7 +49,17 @@ public TPSPinResetProcessor(TPSSession session) {
 
     @Override
     public void process(BeginOpMsg beginMsg) throws TPSException, IOException {
-        if (beginMsg == null) {
+
+        IConfigStore configStore = CMS.getConfigStore();
+        String configName = "op.pinReset.testNoBeginMsg";
+        boolean testPinResetNoBeginMsg = false;
```

Comment:
please add a comment to state the purpose as as "// for testing only; do not use in operation"